src/momentintime.jl

```diff
@@ -119,6 +155,22 @@ Construct an [`MIT`](@ref) instance from `year` and `period`. This is valid only
 for frequencies subtyped from [`YPFrequency`](@ref).
 """ 
 MIT{F}(y::Integer, p::Integer) where F <: YPFrequency{N} where N = MIT{F}(N * Int(y) + Int(p) - 1)
+function MIT{F}(y::Integer, p::Integer) where F <: Weekly{N} where N 
```

Comment:
Something seems wrong with the constructor for `MIT{Weekly{N}}`. 
```julia
julia> MIT{Weekly{5}}(2022,1)
2022W1

julia> MIT{Weekly{6}}(2022,1)
2021W52

```
And `MIT{Weekly}(2022,1)` gives me an error.

Reply:
I should probably improve the docstring for this.